pkg/http/api_directmessaging.go

```diff
@@ -198,7 +198,7 @@ func (a *api) onDirectMessage(w http.ResponseWriter, r *http.Request) {
 // 1. HTTP header 'dapr-app-id' (path is method)
 // 2. Basic auth header: `http://dapr-app-id:<service-id>@localhost:3500/<method>`
 // 3. URL parameter: `http://localhost:3500/v1.0/invoke/<app-id>/method/<method>`
-func findTargetIDAndMethod(reqPath string, headers http.Header) (targetID string, method string) {
+func findTargetIDAndMethod(reqPath string, headers http.Header) (string, string) {
```

Comment:
Why was this changed? It seems functionally equivalent in all ways?

Reply:
We should avoid named return variables at all costs (imo), and _especially_ never do naked returns. The linter was complaining to me about the naked returns.

Reply:
Ok on not using naked returns (agree). I do like named returns here because they're both strings and using named returns helps documenting the signature of the method.